backend/tracim_backend/migration/versions/bfbd0fa4d8a9_workspace_descriptions_to_html.py

```diff
@@ -0,0 +1,83 @@
+"""workspace_descriptions_to_html
+
+Revision ID: bfbd0fa4d8a9
+Revises: 208eda5a6a80
+Create Date: 2021-04-14 14:31:00.007292
+
+Note(RJ): This migration converts plain text descriptions to HTML
+by replacing new lines to <br /> tags. This is done because previously,
+the frontend did this transformation when displaying the descriptions.
+Descriptions are now written using a WYSIWYG editor and we had to remove
+this behavior to make this work.
+
+Workspace descriptions need to always be sent as HTML code from now on.
+"""
+import html
+
+from alembic import op
+from bs4 import BeautifulSoup
+import sqlalchemy as sa
+from sqlalchemy import MetaData
+from sqlalchemy.ext.declarative import declarative_base
+
+NAMING_CONVENTION = {
+    "ix": "ix_%(column_0_label)s",
+    "uq": "uq__%(table_name)s__%(column_0_name)s",  # Unique constrains
+    # for ck contraint.
+    # "ck": "ck_%(table_name)s_%(constraint_name)s",
+    "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+    "pk": "pk_%(table_name)s",
+}
+
+metadata = MetaData(naming_convention=NAMING_CONVENTION)
+DeclarativeBase = declarative_base(metadata=metadata)
+
+# revision identifiers, used by Alembic.
+revision = "bfbd0fa4d8a9"
+down_revision = "208eda5a6a80"
+
+
+class TemporaryWorkspaces(DeclarativeBase):
+    """ temporary sqlalchemy object to help migration"""
+
+    __tablename__ = "workspaces"
+
+    workspace_id = sa.Column(sa.Integer, primary_key=True)
+    description = sa.Column(sa.Text(), unique=False, nullable=False, default="")
+
+
+def nl2br(text: str):
```

Comment:
```suggestion
def newlines_to_html_brtext: str):
```

Reply:
BTW should we handle the case of windows `\r\n` newlines?

Reply:
no, browser don't generate \r\n, even IE on Windows. Copy pasting text should be tested, but the frontend has never handled this hypothetical solution and this is probably safer to emulate what it has been doing